analyzers/tests/SonarAnalyzer.UnitTest/Common/AnalyzerConfigurationTests.cs

```diff
@@ -27,7 +27,6 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 using SonarAnalyzer.Common;
-
```

Comment:
I think it makes sense to separate static namespaces to make them more visible since the behavior is different.